.github/workflows/oppiabot.yml

```diff
@@ -0,0 +1,21 @@
+# Contains jobs corresponding to Oppiabot.
+
+name: Oppiabot
+
+on:
+  pull_request_target:
+    branches:
+      - develop
+
+jobs:
+  oppiabot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: CLA Check
+        uses: oppia/oppiabot@1.2.0
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
```

Comment:
I don't think this is going to work. We can't use secrets like this for forks since they will be exposable to users.